package.json

```
@@ -22,26 +22,29 @@
     "lint:style": "stylelint \"**/*.{js,html}\"",
     "start": "web-dev-server --app-index demo/index.html --node-resolve --open --watch",
     "test": "npm run lint && npm run test:headless",
-    "test:headless": "web-test-runner --files \"./test/**/*.test.js\" --node-resolve",
-    "test:headless:watch": "web-test-runner --files \"./test/**/*.test.js\" --node-resolve --watch"
+    "test:headless": "d2l-test-runner",
+    "test:headless:watch": "npm run test:headless --watch",
+    "vdiff": "npx d2l-test-runner vdiff --chrome",
+    "vdiff:goldens": "d2l-test-runner vdiff golden --chrome",
```

Chrome will run by default:
```suggestion
    "vdiff": "npx d2l-test-runner vdiff",
    "vdiff:goldens": "d2l-test-runner vdiff golden",
```